zynqmp.mk

```diff
@@ -1,8 +1,8 @@
 ROOT			= $(PWD)/..
 PLATFORM		?= zcu102
-PETALINUX_PATH		?= /opt/Xilinx/petalinx_2018_2
+PETALINUX_PATH		?= $(PETALINUX)
 BSP_PATH		?= ./xilinx-${PLATFORM}-v2018.2-final.bsp
-PRJ_PATH		?= $(ROOT)/xilinx-${PLATFORM}-2018.2
+PRJ_PATH		?= $(ROOT)/$(PLATFORM)-$(PETALINUX_VER)
```

Comment:
I had the same question regarding `$(PETALINUX_VER)`, then then I noticed the make target `check-petalinux` which seems to deal with this, so once again I supposed this is a variable stored in `settings.sh`.

Reply:
Yes, i added `check-petalinux` target as dependency for every other targets to be sure that user has actually sourced `settings.sh` before trying to use makefile.